.github/workflows/dapr-e2e-apps.yml

```diff
@@ -12,6 +12,17 @@ on:
       - master
       - release-*
 jobs:
+  pre_job:
+    name: Skip Duplicate Actions
```

Comment:
Please, add condition to only skip in pull requests. Same for the other files, please.